scripts/insertEmailToUniversities.ts

```diff
@@ -59,7 +60,7 @@ const parseFile = () => {
         }),
         {},
       ));
-    insertEmailToUniversities(data);
+    insertEmailToUniversities(data as University[]);
```

Comment:
Can throw exception if cast doesn't work?

Reply:
no, casting is only used on compilation, no runtime exception can be throwns